lego/apps/users/views/membership_history.py

```diff
@@ -27,3 +28,24 @@ def get_queryset(self):
             return queryset.filter(abakus_group__type__in=PUBLIC_GROUPS)
 
         return queryset
+
+    def delete(self, request):
+        try:
+            group = AbakusGroup.objects.get(name=request.data["group_name"])
+            user_membership_history = MembershipHistory.objects.filter(
+                user__id=request.user.id, abakus_group__id=group.id
+            )
+
+            if group.type == GROUP_INTEREST and len(user_membership_history) != 0:
+                name = user_membership_history[0].abakus_group.name
+                user_membership_history.delete()
+                return JsonResponse(
+                    {"result": f"{name} got deleted"}, status=status.HTTP_200_OK
+                )
+            return JsonResponse(
+                {"result": "Nothing to delete"}, status=status.HTTP_400_BAD_REQUEST
+            )
+        except:
+            return JsonResponse(
+                {"result": "Bad Request"}, status=status.HTTP_400_BAD_REQUEST
+            )
```

Comment:
I think we should use `Response`  `rest_framework.response` like we do for the rest of the project to be consistent.

Example for 400 response:

```py
from rest_framework.response import Response
from rest_framework import status

return Response(status=status.HTTP_400_BAD_REQUEST)
```